apps/drec-ui/src/api/device.ts

```diff
@@ -18,3 +23,65 @@ export const useAllDeviceFuelTypes = () => {
 
     return { allTypes, isLoading };
 };
+
+export const useDeviceDetailData = (id: DeviceDTO['id']) => {
+    const { data: device, isLoading: isDeviceLoading } = useDeviceControllerGet(id);
+    const isLoading = isDeviceLoading;
```

Comment:
Why not use isLoading from hook directly?

Reply:
I missed this but not how I missed it